Mathlib/RingTheory/GradedAlgebra/HomogeneousLocalization.lean

```diff
@@ -367,6 +382,8 @@ instance : Add (HomogeneousLocalization 𝒜 x) where
       simp only [num_add, den_add, ← Localization.add_mk]
       convert congr_arg₂ (· + ·) h h' <;> erw [Localization.add_mk] <;> rfl
 
+@[simp] lemma mk_add (i j : NumDenSameDeg 𝒜 x) : mk (i + j) = mk i + mk j := rfl
```

Comment:
Note that, the `Localization` version is called [Localization.add_mk](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/Basic.html#Localization.add_mk). So there is some inconsistencies here (and in `Localization` namespace)

Reply:
I think it should be called `Localization.mk_add_mk` instead (and `Localization.mk_mul` to `Localization.mk_mul_mk`)